src/psyclone/psyGen.py

```diff
@@ -265,24 +265,35 @@ def __str__(self):
 
     @property
     def invokes(self):
+        ''':returns: the list of invokes.
+        :rtype: :py:class:`psyclone.psyGen.Invokes` or derived class
+        '''
         return self._invokes
 
     @property
     def name(self):
+        ''':returns: The name of the PSy object.
```

Comment:
the not The please.

Reply:
One day I will understand capitalisation after : ... I promise.